### PR TITLE
Enable vulnerability scanning in k8s-artifacts-prod

### PR DESCRIPTION
**Pull request by thockin**
Add a group to view results, for now.  Need to find a better way to propagate this and empower each staging group.

### CODE REVIEW

**Comment by thockin** [comment]
I also sorted groups, 1 per commit, so you can review it one commit at a time.

File: groups/groups.yaml [hunk 1]
```diff
@@ -171,6 +171,18 @@ groups:
       - spiffxp@google.com
       - thockin@google.com
 
+  - email-id: k8s-infra-artifact-security@kubernetes.io
+    name: k8s-infra-artifact-admins
```
Comment (listx) [inline groups/groups.yaml:175]: I guess it doesn't matter that the `name` field is the same as line 161?

File: infra/gcp/ensure-prod-storage.sh [hunk 2]
```diff
@@ -146,6 +146,17 @@ upload_gcs_static_content \
...
     "${SCRIPT_DIR}/static/prod-storage"
 
+# Special case: enable vulnerability scanning on the prod GCR.
+enable_api "${PROD_PROJECT}" containerscanning.googleapis.com
+
+# Special case: enable people to read vulnerability reports.
+SEC_GROUP="k8s-infra-artifact-security@kubernetes.io"
+empower_group_as_viewer "${PROD_PROJECT}" "${SEC_GROUP}"
+gcloud \
+    projects add-iam-policy-binding "${project}" \
```
Comment (listx) [inline infra/gcp/ensure-prod-storage.sh:156]: Should this be `${PROD_PROJECT}`?

File: groups/groups.yaml (hunk 1 above)
Comment (spiffxp) [reply groups/groups.yaml:175]:
```suggestion
    name: k8s-infra-artifact-security
```

Comment (thockin) [reply groups/groups.yaml:175]: ACK

File: infra/gcp/ensure-prod-storage.sh (hunk 2 above)
Comment (thockin) [reply infra/gcp/ensure-prod-storage.sh:156]: ACK

**Comment by thockin** [comment]
2 issues fixed and re-pushed

**Comment by spiffxp** [comment]
/lgtm
weird, tide pr dashboard says this PR is all good, but it's not showing up in the [relevant search query](https://github.com/search?utf8=%E2%9C%93&q=is%3Apr%20state%3Aopen%20org%3A%22kubernetes%22%20org%3A%22kubernetes-client%22%20org%3A%22kubernetes-csi%22%20org%3A%22kubernetes-incubator%22%20org%3A%22kubernetes-sigs%22%20-repo%3A%22kubernetes%2Fkubernetes%22%20-repo%3A%22kubernetes%2Fapi%22%20-repo%3A%22kubernetes%2Fapiextensions-apiserver%22%20-repo%3A%22kubernetes%2Fapimachinery%22%20-repo%3A%22kubernetes%2Fapiserver%22%20-repo%3A%22kubernetes%2Fcli-runtime%22%20-repo%3A%22kubernetes%2Fcloud-provider%22%20-repo%3A%22kubernetes%2Fcluster-bootstrap%22%20-repo%3A%22kubernetes%2Fcode-generator%22%20-repo%3A%22kubernetes%2Fcomponent-base%22%20-repo%3A%22kubernetes%2Fcri-api%22%20-repo%3A%22kubernetes%2Fcsi-api%22%20-repo%3A%22kubernetes%2Fcsi-translation-lib%22%20-repo%3A%22kubernetes%2Fkube-aggregator%22%20-repo%3A%22kubernetes%2Fkube-controller-manager%22%20-repo%3A%22kubernetes%2Fkube-proxy%22%20-repo%3A%22kubernetes%2Fkube-scheduler%22%20-repo%3A%22kubernetes%2Fkubectl%22%20-repo%3A%22kubernetes%2Fkubelet%22%20-repo%3A%22kubernetes%2Flegacy-cloud-providers%22%20-repo%3A%22kubernetes%2Fmetrics%22%20-repo%3A%22kubernetes%2Fnode-api%22%20-repo%3A%22kubernetes%2Fsample-apiserver%22%20-repo%3A%22kubernetes%2Fsample-cli-plugin%22%20-repo%3A%22kubernetes%2Fsample-controller%22%20label%3A%22lgtm%22%20label%3A%22approved%22%20label%3A%22cncf-cla%3A%20yes%22%20-label%3A%22do-not-merge%22%20-label%3A%22do-not-merge%2Fblocked-paths%22%20-label%3A%22do-not-merge%2Fcontains-merge-commits%22%20-label%3A%22do-not-merge%2Fhold%22%20-label%3A%22do-not-merge%2Finvalid-commit-message%22%20-label%3A%22do-not-merge%2Finvalid-owners-file%22%20-label%3A%22do-not-merge%2Frelease-note-label-needed%22%20-label%3A%22do-not-merge%2Fwork-in-progress%22%20-label%3A%22needs-rebase%22)

**Comment by spiffxp** [comment]
guess that did the trick

**Comment by spiffxp** [comment]
updated groups

**Comment by spiffxp** [comment]
Ran `./infra/gcp/ensure-prod-storage.sh`

**Comment by spiffxp** [comment]
Updated https://github.com/kubernetes/k8s.io/pull/714 with another commit represent what changed when I ran scripts for this PR.

A bit more than I expected? I guess some previous changes must have only been partially applied

**Pull request merged**